pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -339,6 +339,10 @@ buildStdenv.mkDerivation {
     + lib.optionalString (lib.versionAtLeast version "134") ''
       sed -i "s/icu-i18n/icu-uc &/" js/moz.configure
     ''
+    # This entry in the branding file will override our environment variable, if we let it.
+    + lib.optionalString (branding == "browser/branding/aurora") ''
+      sed -i '/^MOZ_APP_REMOTINGNAME=/d' browser/branding/aurora/configure.sh
+    ''
     + extraPostPatch;
```

Comment:
Why here and not in `extraPostPatch`? This looks very devedition specific right now.

Reply:
It is. I put it here because that's where the rest of the `MOZ_APP_REMOTINGNAME`-setting code is. I initially had this sed apply to all of the branding files (before realizing that `aurora` was the only one with a matching entry), but I restricted it in order not to churn the other derivations unnecessarily. `extraPostPatch` would work just as well; I'll just have to put a little more detail in the comment about it, I think.